tests/datalab/issue_manager/test_utils.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pytest
+
+# from scipy.sparse import csr_matrix
+
+from cleanlab.datalab.internal.issue_manager.utils import ConstructedKNNGraph
+from cleanlab.datalab.datalab import Datalab
+
+
+class TestConstructedKNNGraph:
+    @pytest.fixture
+    def datalab_instance(self):
```

Comment:
You can remove this fixture and instead use the `lab` fixture from [conftest.py.](https://github.com/cleanlab/cleanlab/blob/0903c2a3614aa59dea6b826dfacbc995d5bf9c45/tests/datalab/conftest.py#L58).